pyboy/plugins/game_wrapper_super_mario_land.py

```diff
@@ -30,6 +97,8 @@ class GameWrapperSuperMarioLand(PyBoyGameWrapper):
     If you call `print` on an instance of this object, it will show an overview of everything this object provides.
     """
     cartridge_title = "SUPER MARIOLAN"
+    tiles_compressed = tiles_compressed
+    tiles_minimal = tiles_minimal
```

Comment:
Do we have tests covering these?

Reply:
Nope, same that for previous problem

Reply:
I guess it's ok only to test that code is working. We don't need to exhaustively test that every classification is correct.